### DISCUSSION TITLE
remove port number requirement for mssql

### DISCUSSION THREAD

**Discussion by bloukanov** [sfu-db/connector-x#248] [Ideas]
hi again!

ConnectorX continues to be a central piece to our pipeline for downloading SQL to disk.

Would it be possible to remove the port number requirement for MS SQL? so all that is required is `host%5Cinstance/db?trusted_connection=True`? port is not required for `pyodbc` or `pandas`.

thanks for consideration!

**Comment by wangxiaoying** [comment]
Hi @bloukanov , currently the port value is not required in connectorx. For example the connection string can be look like: `mssql://username:password@host/db`. We will use the default value 1433 in this case.

**Reply by bloukanov** [reply]
Ah ok but our port number is not 1433. I tried and this does not work. perhaps is MS SQL the `instance` in `host\instance` specifies a unique port?
